api/models/issue_map.js

```diff
@@ -0,0 +1,36 @@
+import { ObjectId } from "mongodb";
+import mongoose from "mongoose";
+
+const Schema = mongoose.Schema;
+
+//issue map schema
+const IssueMapSchema = new Schema(
+  {
+    ObjectId: { type: ObjectId, required: true, unique: true },
+    issueNumber: { type: Number, required: true },
+    issueName: { type: String, required: true, unique: true },
+    sections: {
```

Comment:
Sections should be a SubDocument. Defined the sections schema in this file and reference it here as a list of Section, as in:

```sections: [SectionSchema]```